src/native/corehost/runtime_config.cpp

```diff
@@ -396,15 +396,16 @@ bool runtime_config_t::read_framework_array(const json_parser_t::value_t& framew
 
 bool runtime_config_t::ensure_parsed()
 {
-    trace::verbose(_X("Attempting to read runtime config: %s"), m_path.c_str());
     if (!ensure_dev_config_parsed())
     {
         trace::verbose(_X("Did not successfully parse the runtimeconfig.dev.json"));
     }
 
+    trace::verbose(_X("Attempting to read runtime config: %s"), m_path.c_str());
     if (!bundle::info_t::config_t::probe(m_path) && !pal::realpath(&m_path, true))
```

Comment:
Should it be just `if (!pal::realpath(&m_path, true))` since that's the prime implicant? i.e. we don't care about whether or not runtimeconfig entry exists in manifest when the real path is valid .

Reply:
If it is in the bundle, the json parser will read it from there (even if the file also exists on disk), so we want the manifest check. Or am I missing what you are saying?

Reply:
Ah, makes sense. I didn't realize that bundle embeds runtimeconfig.json file as well. So the current condition suggests that in case of bundle, we support both embedded and physical runtimeconfig.json.

Reply:
Yeah, the bundle supports both with embedded taking precedence.